picard/ui/options/interface.py

```diff
@@ -36,7 +37,22 @@ class InterfaceOptionsPage(OptionsPage):
     PARENT = "advanced"
     SORT_ORDER = 40
     ACTIVE = True
-
+    SEPARATOR = u'—'*5
+    TOOLBAR_BUTTONS = {'add_directory_action': {'label': N_(u'Add Folder'), 'icon': 'folder'},
```

Comment:
Nitpicking mode on: i would reformat the dict:

```python
TOOLBAR_BUTTONS = {
    'add_directory_action': {
        'label': N_(u'Add Folder'),
        'icon': 'folder'
    },
    'add_files_action': {
        'label': N_(u'Add Files'),
        'icon': 'document-open'
   },
...
```

Using one element per line makes it easier to diff (and review future changes).